frontend/pages/SoftwarePage/components/cards/SoftwareDetailsSummary/SoftwareDetailsSummary.tsx

```diff
@@ -13,11 +13,19 @@ import DataSet from "components/DataSet";
 import LastUpdatedHostCount from "components/LastUpdatedHostCount";
 import TooltipWrapper from "components/TooltipWrapper";
 import CustomLink from "components/CustomLink";
+import Button from "components/buttons/Button";
+import Icon from "components/Icon";
+import GitOpsModeTooltipWrapper from "components/GitOpsModeTooltipWrapper";
 
 import SoftwareIcon from "../../icons/SoftwareIcon";
 import OSIcon from "../../icons/OSIcon";
 
-const baseClass = "software-details-summary";
+/** This class are re-used on a edit icon modal > preview */
+export const baseClass = "software-details-summary";
+/** This class are re-used on a edit icon modal > preview */
+export const infoClass = `${baseClass}__info`;
+/** This class are re-used on a edit icon modal > preview */
+export const descriptionListClass = `${baseClass}__description-list`;
```

Comment:
im a bit confused about the exporting of these classes to be used on an other component. Are we doing this cause there is some reused styles and markdown between the two? if that is the case would it make sense instead to create a component that is used instead of having only the classes shared.

Reply:
ok, I think this is the best I can get with the time constraints, check it out.